### DISCUSSION TITLE
Update 7/11/2023

### DISCUSSION THREAD

**Discussion by PoohBear2** [deepskies/deeplenstronomy#136] [General]
Today, me and Sofia both met with Brian and his team to update them on our findings. I wrapped up my literature search concerning how gravitational lensing images are detected, and have begun to read into ways that analysis programs use these detected images to identify the deep lensing images within them. I am stuck on one question, however. Where is the lensing kernel within Deeplenstronomy? How do the yaml textfiles that me and Sofia are writing now affect it in any way? How do we affect the kernel through the yaml textfiles, or through some other means? (I will make a github question, but for now, I am looking into it and want to get some background before I propose them.) As a side note, kernel optimization is one the fundamental optimizations that CNNs are designed to perform… Maybe there is a CNN out there that serves this specific purpose? What do you think? I also made some more minor edits to the redshift, none of which, as of now, have made any significant difference to the image resolutions. I will summarize them into a grid once testing is over like you wanted.

**Comment by bnord** [comment]
For the redshift, one way to think of the kernel is as the difference between the lensing object and the source object. so, what is the range of lens redshifts and source redshifts to be used to get a good image?
